web/components/data/FcViewDataAggregate.vue

```diff
@@ -265,12 +266,26 @@ export default {
   },
   methods: {
     async actionDownloadReportFormatCollisions(reportFormat) {
-      const job = await postJobGenerateCollisionReports(
-        this.auth.csrf,
-        this.locationsSelection,
-        this.filterParamsCollision,
-        reportFormat,
-      );
+      let job;
+      if (this.locationsSelection.selectionType === LocationSelectionType.CORRIDOR) {
+        this.locationsSelectionForCorridorReport = JSON.parse(
```

Comment:
I dont love this but I need a deep copy of the locationsSelection object and this was just the easiest to implement. I guess if we need to do this somewhere else, it would make sense to create a utility function that does the cloning a bit more efficiently.